internal/olm/operator/uninstall.go

```diff
@@ -143,9 +149,13 @@ func (u *Uninstall) Run(ctx context.Context) error {
 		return err
 	}
 	var objs []client.Object
+
 	if u.DeleteCRDs {
 		objs = append(objs, crds...)
+	} else {
+		u.Logf("skipping deletion of CRDs as delete-crds flag is set to %v", u.DeleteCRDs)
```

Comment:
```suggestion
		u.Log("Skipping CRD deletion")
```

Reply:
Would be nice to have a log statement for operator groups as well, if one doesn't exist already.